operator/internal/central/extensions/reconcile_pvc.go

```diff
@@ -36,72 +39,133 @@ const (
 	// PVCTargetCentral is for any PVC that would be attached to the Central deployment
 	PVCTargetCentral PVCTarget = "central"
 
-	// PVCTargetCentralDB is for any PVC that would be attached to the Central DB deployment
+	// PVCTargetCentralDB is for a PVC that would be attached to the Central DB
+	// deployment as a data volume
 	PVCTargetCentralDB PVCTarget = "central-db"
+
+	// PVCTargetCentralDBBackup is for a PVC that would be attached to the
+	// Central DB deployment as a backup volume
+	PVCTargetCentralDBBackup PVCTarget = "central-db-backup"
 )
 
 var (
 	errMultipleOwnedPVCs = errors.New("operator is only allowed to have 1 owned PVC")
 
-	defaultPVCSize = resource.MustParse("100Gi")
+	DefaultPVCSize       = resource.MustParse("100Gi")
+	DefaultBackupPVCSize = resource.MustParse("200Gi")
 )
 
-func convertDBPersistenceToPersistence(p *platform.DBPersistence) *platform.Persistence {
+func convertDBPersistenceToPersistence(p *platform.DBPersistence,
+	target PVCTarget, log logr.Logger) (*platform.Persistence, error) {
```

Comment:
TBH I'm finding the unbroken function signatures much easier to read, given the indentation of the second line is the same as the body's.
Looking at existing cases, the length does not seem to be triggering any linter errors, so would you mind unbreaking the line here and elsewhere?